spec/components/Web3API_Client.md

```diff
@@ -0,0 +1,9 @@
+# Web3API Client
+
+Web3API client is intended for application developers that can use this single library to interact with various decentralized services.
+
+Currently, Web3API client is available in JavaScript language.
```

Comment:
maybe 
```
Web3API is available for following programming languages:
- Javascript/Typescript
- TBD
```

Reply:
Yep, list is more better. Rewriting this now, will be in the client PR